gax-httpjson/src/test/java/com/google/api/gax/httpjson/RetryingTest.java

```diff
@@ -328,8 +352,9 @@ public void testUnknownStatusCode() {
         HttpJsonCallableFactory.create(callInt, callSettings, clientContext);
     try {
       callable.call(1);
-    } catch (HttpJsonApiException exception) {
-      Truth.assertThat(exception.getStatusCode().getCode()).isEqualTo(STATUS_UNKNOWN);
+    } catch (UnknownException exception) {
+      Truth.assertThat(((HttpJsonStatusCode) exception.getStatusCode()).getCode())
+          .isEqualTo(STATUS_UNKNOWN);
```

Comment:
In the cases where we catch a specific exception, we no longer need to check the status codes.

Reply:
Done.